app/views/docs/configuration.phtml

```diff
@@ -0,0 +1,65 @@
+<p>
+    Self-hosting allows you to customize Appwrite to meet your needs and preferences. Here's a list of things you will need to setup before developing with Appwrite. Before you start, backup your <code>.env</code> and <code>docker-compose.yml</code> files.
+</p>
+
+<h2><a href="#localhost" id="localhost">Localhost Development</a></h2>
+<p>Self-hosting Appwrite for development lets you quickly prototype on your local machine. Since localhost is only a loopback address that points back at your development machine, there are some caveats you should be aware of when working with localhost.</p>
+<p>Appwrite generates a self-signed certificate for iOS and Android development on localhost, as they don't allow unsecure HTTP by default. If you're developing Android or iOS apps on localhost, set <code>setSelfSigned(true)</code> while initializing your Appwrite client.</p>
+<p>If you're using an Android emulator to test your Flutter or Android apps, you can access localhost of your machine using the special alias IP <code>10.0.2.2</code>.</p>
```

Comment:
Im not sure 10.0.2.2 is true for all cases

Reply:
https://developer.android.com/studio/run/emulator-networking

Found it here. I think it's meant to be a feature for android emulators

Reply:
Okay then 👍  sounds good